### ISSUE TITLE
No support for bugs at all?

### ISSUE THREAD

**Issue by tomasts248** [NickPiscitelli/Glider.js#216]
Sad but seems like it.

**Comment by NickPiscitelli** [comment]
There are no critical bugs that are open right now.

Happy to review any PRs fixing known bugs.

Please don't open patronizing issues on GitHub.

**Comment by tomasts248** [comment]
This is the second bug I have had with Glider.js similar as before it took ages for you to reply to it. I mean I understand it is a standalone project and I'm thankful for it as well as being open source but it's been months and mean while part of the website functionality is broken :( What I'm supposed to do?

**Comment by NickPiscitelli** [comment]
I did not request that you use this software. You are far from thankful. If you are interested in the project, consider contributing to it.

**Comment by tomasts248** [comment]
Then why offering in the first place if is not for others to be used? amazing logic, but no worries ill will certainly avoid it in the future and won't recommend it any longer, alienating one dev at a time good luck.

**Comment by NickPiscitelli** [comment]
All of your reported issues were implementation problems on your end. On top of the free plugin, you are looking for free tech support and then complaining it wasn't quick enough for you.

It's "developers" like you who make us regret ever contributing to open source.

In the future, please reconsider your approach in situations like these. No open source projects owe you anything. Life happens and sometimes we don't have the time to dedicate to a project. Then when you do find the time to log on, you are met by entitled individuals chastising you for not answering them soon enough.

**Comment by tomasts248** [comment]
> All of your reported issues were implementation problems on your end.

Even the one you fixed before?

> No open source projects owe you anything. 

Well I strongly disagree, If you want I could further argue perhaps we both learn something but for now consider that I got people complaining on my end for those problems as well, I even offered to pay other devs to fix those, and they couldn't.
You did had the time to quickly reply to this thread but not for reading the other issues no only by me but others users of  your slider wich is sad, it doesn't take much to reply does it?

**Comment by NickPiscitelli** [comment]
> You did have the time to quickly reply to this thread but not for reading the other issues

I have time now, I do not always have time. Why imply such a silly concept? If you would bother to check, you would notice I am using this time to reply to other users and their issues. Unfortunately some of that time was consumed by your unprofessional comments. 

> Well I strongly disagree, If you want I could further argue perhaps we both learn something 

I would truly love to hear why any open source project owes you anything.

**Comment by tomasts248** [comment]
> Unfortunately some of that time was consumed by your unprofessional comments.

Yeah lot's of time by the way, but okey that point is of no importance as you pointed.

> I would truly love to hear why any open source project owes you anything.

Okey does Mozilla Firefox owe nothing to its users? same for Wordpress to the devs that choose to work with it? I think they do, beyond the license terms...

**Comment by NickPiscitelli** [comment]
> Yeah lot's of time by the way

You really don't know when to stop, always antagonizing. I spent no more than an hour arguing with you, but you are correct that is has been of no importance. Some of that time was actually spent trying to triage your problems, but they were reported incorrectly so you wasted more of my time.

> Okey does Mozilla Firefox owe nothing to its users?

Correct, they do not owe their users anything. Their users should be grateful that such wonderful products are released for free. If they owe anyone anything, it's their donors; certainly not their users.

I'm no longer replying to these threads and to be honest, I would be glad if you stopped using this plugin. I do not appreciate your complete lack of diplomacy and professionalism.

**Comment by tomasts248** [comment]
> Correct, they do not owe their users anything. Their users should be grateful that such wonderful products are released for free. If they owe anyone anything, it's their donors; certainly not their users.

That explains everything, quoting a friend:" if you want your slider to be successful documentation and/or support of both end-users and developers is absolutely key" Of course you don't seem to care much for this.

Said that, yes the way I expressed myself wasn't the best, but on the other hand If I didn't post this thread pretty sure you would had replied even later (three months is a lot). I regret so much choosing your plugin because of this I invested on it, and of course there are very competent alternatives.

> I'm no longer replying to these threads

"Silence grants"
